packages/inferno/inferno.20190924/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "francois.pottier@inria.fr"
+authors: [
+  "François Pottier <francois.pottier@inria.fr>"
+]
+homepage: "https://gitlab.inria.fr/fpottier/inferno"
+dev-repo: "git+https://gitlab.inria.fr/fpottier/inferno.git"
+bug-reports: "francois.pottier@inria.fr"
+license: "MIT"
+synopsis: "A library for constraint-based Hindley-Milner type inference"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" { >= "4.02.3" }
+  "dune"  { build }
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
According tot he dune-project file in the repository you require at least 1.11. Since a few months dune can no longer be marked as build dependency.